kustomize/monitoring/prometheus-config.yaml

```diff
@@ -33,19 +43,25 @@ data:
         target_label: kubernetes_namespace
       - source_labels: [__meta_kubernetes_pod_name]
         target_label: pod
-      - source_labels: [__meta_kubernetes_namespace,__meta_kubernetes_pod_label_postgres_operator_crunchydata_com_cluster]
+      - source_labels: [__meta_kubernetes_pod_label_postgres_operator_crunchydata_com_cluster,__meta_kubernetes_pod_label_pg_cluster]
+        target_label: cluster
+        separator: ""
+        replacement: '$1'
+      - source_labels: [__meta_kubernetes_namespace,cluster]
         target_label: pg_cluster
-        separator: ':'
+        separator: ":"
         replacement: '$1$2'
```

Comment:
🤔 Am I reading this correctly? v5 `cluster` and v4 `pg_cluster` are merged into `cluster` then `pg_cluster` is created from namespace and `cluster`.

Should we drop `cluster` after this?

🤔 Is it possible to do it without the intermediate label?